sdks/csharp/sdk/AgonesSDK.nuspec

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package >
+  <metadata>
+    <id>AgonesSDK</id>
+    <version>1.0.4</version>
```

Comment:
This was arbitrarily picked, feel free to pick a starting version.

Reply:
Should align with next Agones version - so 1.7.0